analyses/metacells/README.md

```diff
@@ -9,20 +9,38 @@ Initially, this module will contain some scripts and notebooks for applying anal
 
 ## Usage
 
-TBD
+To run the module, first install and activate the `openscpca-metacells` conda environment.
+From the module directory, run:
+
+```sh
+conda-lock install --name openscpca-metacells
+conda activate openscpca-metacells
+```
+
+Then run the main bash script:
+
+```sh
+bash run-metacells.sh
+```
+
 
 ## Input files
 
 Initial analysis is expected use processed `SingleCellExperiment` objects from the ScPCA Portal, in AnnData format.
 
+For initial testing, files were downloaded with:
```

Comment:
Just confirming this code is right - since you wrote _testing_, I wondered if you meant to include the test flag? If not, then not!